lib/senkyoshi/models/question_bank.rb

```diff
@@ -0,0 +1,41 @@
+require "senkyoshi/models/qti"
+require "byebug"
+
+module Senkyoshi
+  class QuestionBank < QTI
+    def canvas_conversion(course, resources)
+      question_bank = CanvasCc::CanvasCC::Models::QuestionBank.new
+      question_bank.identifier = @id
+      question_bank.title = @title
+      question_bank = setup_question_bank(question_bank, resources)
+      course.question_banks << question_bank
+      course
+    end
+
+    def setup_question_bank(question_bank, resources)
```

Comment:
From what I could see in Canvas doesn't have descriptions in the question banks nor does the canvas_cc gem so I am guessing it might not be useful.